### DISCUSSION TITLE
Is there an equivalent to TextChannel.archived_threads() for ForumChannel?

### DISCUSSION THREAD

**Discussion by Tabris05** [Rapptz/discord.py#8980] [Q&A] [answered]
Title. Making a bot that posts in forum threads that could very well become archived before the next time the post event is triggered and can't find a way to get thread objects for archived threads.

**Comment by LostLuma** [comment, marked as answer]
Hello, this specific method was added for forums recently in #8476.

If you want to fetch only a single specific thread you already know the ID of you can use [`Guild.fetch_channel`](https://discordpy.readthedocs.io/en/latest/api.html#discord.Guild.fetch_channel) to retrieve it, otherwise the `archived_threads` method will be included in the 2.1 release.

To use it early you can install discord.py from the development branch instead of pip:

```sh
# Latest commit
pip install -U "discord.py[speed] @ git+https://github.com/Rapptz/discord.py@master"
# Locked to a specific commit from master (the latest currently)
pip install -U "discord.py[speed] @ git+https://github.com/Rapptz/discord.py@a592361f29ba31f08f43fa60ed3685905e5da329"
```

You can include the quoted parts inside your requirements.txt if you'd like, however it's recommended to lock it to a specific commit instead of the branch as there may (in the far future) be breaking changes on the branch again.

**Reply by Tabris05** [reply]
Will do, thank you!
